azkaban-common/src/main/java/azkaban/imagemgmt/models/ImageRampRule.java

```diff
@@ -0,0 +1,50 @@
+package azkaban.imagemgmt.models;
+
+import java.util.List;
+import java.util.Set;
+
+
+public class ImageRampRule extends BaseModel {
+  // ramp rule name
+  String ruleId;
+  // image name of ramp rule for
+  String imageName;
+  // image version of ramp rule for
+  String imageVersion;
+  // rule owners
+  Set<String> owners;
+  // Ramp Rule for HP flows or not
```

Comment:
Change the comment to suggest what the value implies.
Ramp Rule for HP flows. True if the flow is HP, false otherwise.

Reply:
updated.